test-utils/src/utils/ProxyCacheTestData.scala

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import play.api.libs.json.Json
+import uk.gov.hmrc.statepension.domain.nps.{Liabilities, NIRecord, ProxyCacheData, Summary}
+
+object ProxyCacheTestData {
+  val niRecordJson: String =
```

Comment:
Looking at the Json here. It looks like the test is going to contain fields which won't be passed on by the proxy-cache. Is it worth swapping this out for something that better represents the payload from the proxy-cache?